src/hive/components/screens/errors.cljs

```diff
@@ -46,7 +46,7 @@
        [:> react/TouchableOpacity
          {:style {:borderRadius 5 :backgroundColor "red" :height 40 :width 60
                   :justifyContent "center" :alignItems "center"}
-          :on-press goBack}
+          :on-press #(goBack)}
```

Comment:
What was the problem with `goBack`? Isn't it a function?

Reply:
@mehdisadeghi I went with the inertia from Clojure. In Clojure, static functions cannot be used directly as arguments since they are not Clojure Function objects. I dont remember if it is the same on Cljs 😅